cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -167,7 +169,11 @@ public static void editConfig() throws Exception {
                 System.out.println(
                         "\n-New  configuration file will be created at: " + configFile.getAbsolutePath() + "\n");
                 configFile.createNewFile();
-            }
+            }/*else{
```

Comment:
no need to commit commented out code like this.

Reply:
will remove these